### PR TITLE
Add 2024.8 compatibility

### PR DESCRIPTION
**Pull request by piitaya**
Hi 👋

The Home Assistant frontend will have some breaking changes in `2024.8`. As there is only few custom features, I prefer to do PR for migration instead of supporting legacy config in Home Assistant frontend.

`customTileFeatures` has been renamed to `customCardFeatures` since `2023.12`. Both variables are supported until 
`2024.8` so I renamed it in this PR.

Also, the default height of features has been adjusted from 40px to 42px. The 42px is set using `--feature-height` css variable so it's backward compatible with version < `2024.8`.

Finally, the feature padding is not set by the card and not the feature itself, so I removed if version is at least `2024.8`.

This version should be valid before and after `2024.8`.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export const atLeastHaVersion = (
```
Comment (Nerwyn) [inline src/utils.ts:1]: Please rename file and move into utils folder after changing PR target branch to dev.

**Comment by piitaya** [comment]
Ok, I rebased the changes to the dev branch 🙂

**Comment by Nerwyn** [comment]
Thanks! I'll do a final review and merge after work.

File: src/utils/atLeastHaVersion.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+export const atLeastHaVersion = (
```
Comment (Nerwyn) [inline src/utils/atLeastHaVersion.ts:1]: Export this from `utils/index.ts` and import it from `utils` above to be consistent with the deepKeys export/import.

**Comment by Nerwyn** [comment]
@piitaya any idea why the [HACS action validation](https://github.com/Nerwyn/service-call-tile-feature/actions/runs/10009874475/job/27684294269#step:4:31) is failing?

```
##[debug] <Plugin piitaya/service-call-tile-feature> Running checks against 2024-8-compatibility
##[debug] 'GitHubResponseHeadersModel' is missing key 'x_accepted_github_permissions' for <class 'str'>
Error:  <Plugin piitaya/service-call-tile-feature> Repository structure for 2024-8-compatibility is not compliant
```

I'm not entirely sure what the above means and why my repo isn't throwing it but your fork is.

**Review by Nerwyn — APPROVED** [review]

**Pull request merged**